markdown/dev/reference/api/stack/home/en.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Stack.home()
+---
+
+The `Stack.home()` method calculates the stack's bounding box, sets it,
```

Comment:
change "sets it" to, "setting the `width`, `height`, `topLeft` and `bottomRight` properties",

Reply:
Changed as suggested.